website/common/locales/en/groups.json

```diff
@@ -132,7 +132,14 @@
   "possessiveParty": "<%= name %>'s Party",
   "clearAll": "Delete All Messages",
   "confirmDeleteAllMessages": "Are you sure you want to delete all messages in your inbox? Other users will still see messages you have sent to them.",
-  "optOutPopover": "Don't like private messages? Click to completely opt out",
+  "PMEnable": "Enable",
+  "PMEnableFull": "Enable private messages",
```

Comment:
Can you please capitalise the first letters in "Private Messages" here and in the other strings below? This is a relatively recent convention in Habitica.